frontend/public/components/dashboard/project-dashboard/status-card.tsx

```diff
@@ -1,22 +1,56 @@
 import * as React from 'react';
+// FIXME upgrading redux types is causing many errors at this time
+// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+// @ts-ignore
```

Comment:
Why we need this?

Reply:
this is not needed anymore. removed. Thanks.